python/arcticc/pb2/__init__.py

```diff
@@ -0,0 +1,13 @@
+import os as _os
+import sys as _sys
+import google.protobuf as _protobuf
+
+_proto_ver = _protobuf.__version__.split(".")[0]
+if _proto_ver in "34":
+    # Use the namespace package (https://peps.python.org/pep-0420) feature to select the pb2 files matching the protobuf
+    _protos_path = _os.path.abspath(
+        _os.path.join(_os.path.dirname(__file__), "..", "..", "arcticdb", "proto", _proto_ver, "arcticc", "pb2")
```

Comment:
I am a bit worried about this. What if `arcticdb` and `arcticc` are not installed in the same directory?

For example, when someone is working on an [editable install](https://peps.python.org/pep-0662/) of `man.arcticdb` (or whatever we call it) and executed its `arcticc/pb2/__init__.py` first (which adds a non-existant path to `os.path`), will the `__init__.py` from `arcticdb`'s wheel be called again?

Reply:
I think this will be the _only_ `arcticc/pb2` package in the codebase, `man.arcticdb` won't have one.